### PR TITLE
Adds remote enrichment support

### PR DESCRIPTION
**Pull request by petterik**
Implements existing upload_attributes
and export_dataset stubs with real
functionality.

### CODE REVIEW

**Review by ryttry — APPROVED** [review]

**Comment by petterik** [comment]
@ryttry I updated this PR to validate successful summary from the update_attributes endpoint call based on the discussions in the backend PR.

Merge this PR if you think it looks good!

File: src/watchful/enrich.py [hunk 1]
```diff
@@ -366,9 +364,7 @@ def main(args: List[str] = None, custom_enricher: Enricher = None) -> None:
     # Upload created attributes output file to Watchful application if it is
     # remote.
     else:
-        client.upload_attributes(
-            dataset_id, dest_attr_filename, args.out_file, project_id
-        )
+        client.upload_attributes(dataset_id, args.out_file)
```
Comment (ryttry) [inline src/watchful/enrich.py:367]: Suggest `_ = client.upload_attributes(dataset_id, args.out_file)` so we easily know there is information returned there that may be used in future.

Comment (ryttry) [reply src/watchful/enrich.py:367]: @petterik LGTM, just one comment above. Have you run the api tests?

Comment (petterik) [reply src/watchful/enrich.py:367]: Yes! I had some failures but those have been resolved now. I'll merge this and create a new release

**Pull request merged**